app/src/main/java/com/github/mateo762/myapplication/ui/home/TodayScreen.kt

```diff
@@ -306,14 +312,25 @@ fun getNextUpLocalDateTime(
 }
 
 @RequiresApi(Build.VERSION_CODES.O)
-fun parseTime(timeString: String): LocalTime {
+private fun parseTime(timeString: String): LocalTime {
     val components = timeString.split(":").map { it.toInt() }
     return LocalTime.of(components[0], components[1])
 }
 
 @RequiresApi(Build.VERSION_CODES.O)
-fun minutesUntilStartTime(startTime: String, now: LocalTime): Long {
+private fun minutesUntilStartTime(startTime: String, now: LocalTime): Long {
     val start = parseTime(startTime)
     val duration = Duration.between(now, start)
     return duration.toMinutes()
+}
+
+private fun Context.findFragmentActivity(): FragmentActivity? {
```

Comment:
Also, what does this do? / Where is it used?

Reply:
It's used in that class (TodayScreen), it is used to open the UploadFragment